README.md

```diff
@@ -125,11 +125,16 @@ Command Line Options:
   - -b Bidirectional data: Data will be generated in both directions
      of a TCP stream. ACKs will be turned on.  This feature is off
      by default.
+
   - -B Background Traffic Percentage: Set this value between 1 and
      100 to produce Background Traffic. This traffic will consist of
      even selections of following generic application protocols:
      FTP, HTTP, IMAP, POP and SMTP. By default it is set to 0.
 
+  - -B [Background Traffic Protocol:Percentage]: Set at least one
```

Comment:
First, How about the idea distributing the total background traffic percentage separately? like
-B http:20,ftp:30,pop:10
If there is no problem, I would like to write the usage as follows.
```
- -B [Background Traffic Protocol:Percentage]: Set at least one
protocol with value between 1 and 100 to produce Background Traffic.
This value represents the percentage of the total amount of traffic.
The following protocols are available: FTP, HTTP, IMAP, POP and SMTP.
For example: "http:20,ftp:30,smtp:10".
Enter only one number as the argument value to generate randomly 
 traffic .
For example: "80".
```
how about this?